Dockerfile

```diff
@@ -28,4 +28,8 @@ RUN install -s runc /usr/local/bin
 RUN runc --version
 
 FROM scratch
+ARG GO_IMAGE
+LABEL org.opencontainers.image.url="https://hub.docker.com/r/rancher/hardened-runc"
+LABEL org.opencontainers.image.source="https://github.com/rancher/image-build-runc"
+LABEL org.opencontainers.image.base.name="${GO_IMAGE}"
```

Comment:
is this technically correct? This is a `FROM scratch` image with a single binary in it. Scratch images do not have a base image.

Reply:
Made the changes; Please review

Reply:
@brandond - Can you please review the latest changes applied and let me know your comments.